packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```diff
@@ -173,45 +179,45 @@
         }
         &__row {
             /*                     attribute   standard advanced synthetic */
-            grid-template-columns: 1.7fr       1fr      1fr      1fr;
+            grid-template-columns: 1.7fr 1fr 1fr 1fr;
```

Comment:
This had spaces to align with the comment above it. Perhaps we can modify the comment too (remove spaces and clarify order clearer) or revert the removal.